app/controllers/SummaryDeclarationController.scala

```diff
@@ -125,8 +175,14 @@ class SummaryDeclarationController @Inject() (val mcc: MessagesControllerCompone
     getEntry[CompanyDetailsList](cacheItem, DataKey(ersUtil.SUBSIDIARY_COMPANIES_CACHE))
       .getOrElse(CompanyDetailsList(List[CompanyDetails]()))
 
+  def emptyAltAmends(alterationAmends: AlterationAmends): Boolean =
```

Comment:
This will always return true if the collection exists it should probably be something like this instead:

``` scala
val checkIfEmpty: Boolean = List(altAmendsTerms, altAmendsEligibility, altAmendsExchange, altAmendsVariations, altAmendsOther).forall{_.isEmpty}
```

This is because .exists{_.isDefined} will match if an element in the list contains Some(_). Whereas we're only looking for a list containing all None values.